common/include/common/float_comparison.hpp

```diff
@@ -33,15 +33,15 @@
 #endif
 
 #if SYCL_LANGUAGE_VERSION < 202000
```

Comment:
Probably, we need to decide what to do with this kind of statement.

Is it worth removing them completely since we have broken the backward compatibility changing the selector type?

For the record, `SYCL_LANGUAGE_VERSION` is defined as `202003` in AdaptiveCpp.

Reply:
Since the update to SYCL2020 is not fully completed I thought to keep it, but yes we should discuss it. 
I thought AdaptiveCpp has some way to handle the version since it is not yet fully compatible with SYCL2020, thank you for checking the language version they declare. Does any compiler support `sycl1.2.1` and not sycl2020? We support only DPCPP and AdaptiveCpp, so maybe we can remove it in another PR. 

(@hjabird for knowledge)

Reply:
I would suggest to try and remove the instances of `SYCL_LANGUAGE_VERSION` in this PR as well when possible.
It's not worth trying to support compilers that wouldn't support SYCL 2020 at all.

Reply:
Thank you all. I removed `SYCL_LANGUAGE_VERSION` pragma-check everywhere in 68e76e8.